index.html

```diff
@@ -0,0 +1,578 @@
+  schema-->proc1("🤖 &nbsp;Processor")
+  proc1-->output1(["☉ Core Schema[0]"])
+  output1-->proc2("🤖 &nbsp;Processor")
+  proc2-->output2(["☉ Core Schema[1]"])
+  output2-->etc("...")
+  etc-->final(["☉ Core Schema [final]"])
+  final-->core("🤖 Data Core")
+  schema-->reader("👩🏽‍💻  Reader")
+  output1-->reader
+  output2-->reader
+  final-->reader
+```
+
+- **Authors (either human or machine)** write an initial core schema as specified in this document, including versioned `@core` requests for all directives they use
+- **Machine processors** can process core schemas and output new core schemas. The versioning of directives and associated schema elements provided by the `@core` allows processors to operate on directives they understand and pass through directives they do not.
+- **Human readers** can examine the core schema at various stages of processing. At any stage, they can examine the `@core` directives and follow URLs to the specification, receiving an explanation of the requirements of the specification and what new directives, types, and other schema objects are available within the document.
+- **Data cores** can then pick up the processed core schema and provide some data-layer service with it. Typically this means serving the export schema as a GraphQL endpoint, using metadata defined by machinery to inform how it processes operations it receives. However, data cores may perform other tasks described in the core schema, such as routing to backend services, caching commonly-accessed fields and queries, and so on. The term "data core" is intended to capture this multiplicity of possible activities.
+
+## The Basics
+
+Core schemas:
+  1. **must** be valid GraphQL schema documents,
+  2. **must** contain exactly one `SchemaDefinition`, and
+  3. **should** use the `@core` directive on their schema definition to declare any directives they reference by
+  [`@core(using:)`](#using) a [well-formed specification URL](#spec-urls)
+
+The first `@core` directive on the schema **must** reference the core spec itself, i.e. this document.
+
+<view-of src=basic.graphql title='Basic core schema using @core and @example'></view-of>
+
+### Unspecified directives are passed through
+
+Existing schemas likely contain definitions for directives which are not versioned, have no specification document, and are intended mainly to be passed through. This is the default behavior for core schema processors:
+
+```graphql
+#::[Unspecified directives are passed through]
+schema
+  @core(using: "https://lib.apollo.dev/core/v0.1")
+{
+  query: Query
+}
+
+type SomeType {
+  field: Int @another
+}
+
+# `@another` is unspecified. Core processors will not extract metadata from
+# it, but its definition and all usages within the schema will be exposed
+# in the export schema.
+directive @another on FIELD
+```
+
+### Renaming `@core` itself
+
+It is possible to rename the `@core` directive itself with the same `as` mechanism used for all specs:
```

Comment:
I think this needs to be more specific. What are the precise conditions under which a processor should decide that a particular directive is the `@core` directive?  Is it:

If the first directive used on the `schema` declaration has arguments `using: String!`(*) and `as: String`, and the directive use includes an `as` parameter whose value is equal the name of the directive, and the `using` parameter's prefix is SPEC_URL (**) then that directive is the core directive.

(*) The `!` here is only possible if my suggestion to not combine two distinct forms of `@core` into one directive is taken.
(**) See also my other comment noting that the spec really needs to specify its own SPEC_URL.